plugin/src/main/scala/higherkindness/mu/rpc/srcgen/SrcGenPlugin.scala

```diff
@@ -141,7 +157,10 @@ object SrcGenPlugin extends AutoPlugin {
     muSrcGenCompressionType := NoCompressionGen,
     muSrcGenIdiomaticEndpoints := false,
     muSrcGenOpenApiHttpImpl := HttpImpl.Http4sV20,
-    muSrcGenStreamingImplementation := Fs2Stream
+    muSrcGenStreamingImplementation := Fs2Stream,
+    muSrcGenExecutionMode := Local,
+    muSrcGenCompendiumProtocolIdentifiers := Nil,
+    muSrcGenCompendiumServerUrl := "https://localhost:47047"
```

Comment:
Is the server likely to use HTTPS if it's running on localhost?

Reply:
ok, just checked aaaand nop. At least not by default 😞